plugin/trino-hive/src/test/java/io/trino/plugin/hive/parquet/TestFullParquetReader.java

```diff
@@ -13,11 +13,21 @@
  */
 package io.trino.plugin.hive.parquet;
 
+import org.testng.annotations.Test;
+
+@Test(singleThreaded = true)
 public class TestFullParquetReader
         extends AbstractTestParquetReader
 {
     public TestFullParquetReader()
     {
         super(ParquetTester.fullParquetTester());
     }
+
+    @Test
+    public void dummy()
+    {
+        // required to make @Test(singleThreaded = true) work
+        // issue: https://github.com/cbeust/testng/issues/144
```

Comment:
See https://github.com/trinodb/trino/blob/master/plugin/trino-postgresql/src/test/java/io/trino/plugin/postgresql/TestPostgreSqlCaseInsensitiveMapping.java#L67-L72.

Easier to search and remove consistent things once fixed/version is upgraded.